evm_loader/program/src/account_storage.rs

```diff
@@ -519,6 +519,10 @@ impl<'a> AccountStorage for ProgramAccountStorage<'a> {
         self.find_account(address).is_some()
     }
 
+    fn get_account_solana_address(&self, address: &H160) -> Pubkey {
+        self.get_account(address).unwrap().get_solana_address()
```

Comment:
The transaction will fall with an uninformative message in the case of a missing account in the list of accounts. Maybe we should print an error message containing the account that should have been added to the transaction?